### ISSUE TITLE
2.4.11: Insufficient requirement for focus indication via enlargement

### ISSUE THREAD

**Issue by benja11y** [w3c/wcag#1858]
Figure 15 in the ["Adjacent contrast" section of the Understanding document](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum#Contrast-thickness) shows three buttons which are very close to each-other. The middle button has been enlarged by 2px on each side.

Whilst is looks fairly clear when there are three buttons next to each-other, we do not think that this would be a sufficient focus indicator for many users to perceive if the button was on its own.

This issue has been raised on behalf of [TPGi](https://www.tpgi.com), based on an internal consultation process. [An article containing all of our comments](https://www.tpgi.com/wcag-22-working-draft-feedback/) can be found on our blog, along with an introduction and acknowledgments.

**Comment by alastc** [comment]
Hi @benja11y,

For future, if you have multiple things to say about one SC from one organisation, could you combine them into one issue please?

**Comment by alastc** [comment]
Draft response: 
<hr>
It is a limitation of the SC that some indicators are not as effective when links/buttons are on their own, separate from other elements. For example, switching the background colour is a very obvious change, but if the element is separated from others then it can be harder to notice, particularly for people using magnification.

However, if we don't allow for that, we would be banning other indicators that are generally very obvious. For example:

![image](https://user-images.githubusercontent.com/216630/121608912-b6e84c80-ca4a-11eb-8009-832626df132b.png)
alt=Three green buttons, one has a thick blue border around it.

The contrast between the green and blue is very poor, but generally that indicator (due to thickness) is going to be reasonably visible. The black/black example in the understanding document is highlighting an extreme case.

Also consider that viewing the static image is not the same as watching the focus move around the page. The eye picks up changes, even if they are isolated from other elements. (See this [test page](https://alastairc.uk/tests/wcag22-examples/focus-more-visible-2.html), example 9b).

Overall, whilst it is a limitation, it is a necessary one to prevent creating false-positives, and it is mitigated by the dynamic nature of focus indicators.

**Comment by benja11y** [comment]
@alastc  Noted.

We decided on the approach based on the line “Ideally open one new GitHub issue per discrete comment.” (Found on 
What’s New In WCAG 2.2 https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/).

**Comment by DavidMacDonald** [comment]
Any SC that requires changes to the look and feel of a site requires compromises with visual designers. In this case the allowance of a larger indicator border of the same colour of the button may not be ideal but the consequences of not allowing it may have significant ramifications to designers.

**Comment by alastc** [comment]
Response approved: https://www.w3.org/2021/12/21-ag-minutes.html#t08
